### PR TITLE
gtfs-static-and-rt.jv is added to showcase the processing of GTFS-RT data as well as GTFS data (UAC-3 of #219)

### PR DESCRIPTION
**Pull request by schlingling**
`gtfs-static-and-rt.jv` is added to showcase the processing of GTFS-RT data as well as GTFS data.

This showcases the periodical pull of GTFS and GTFS-RT data using endpoints from french NAP (I have chosen randomly Metropole Brest)
- Summary from NAP for Metropole Brest: https://transport.data.gouv.fr/datasets/horaires-theoriques-et-temps-reel-des-bus-et-tramways-circulant-sur-le-territoire-de-brest-metropole
- GTFS static endpoint: https://ratpdev-mosaic-prod-bucket-raw.s3-eu-west-1.amazonaws.com/11/exports/1/gtfs.zip
- GTFS-RT Alerts endpoint: https://proxy.transport.data.gouv.fr/resource/bibus-brest-gtfs-rt-alerts
- GTFS-RT TripUpdates endpoint: https://proxy.transport.data.gouv.fr/resource/bibus-brest-gtfs-rt-trip-update
- GTFS-RT VehiclePositions endpoint:	https://proxy.transport.data.gouv.fr/resource/bibus-brest-gtfs-rt-vehicle-position

Now we have in total 3 different gtfs-related jayvee-examples:
- `example/gtfs-static.jv` --> Uses the google demo endpoint and showcases how to access GTFS data
- `example/gtfs-rt-simple.jv` --> Uses an endpoint from french NAP and showcases how to access GTFS-RT data (simple
- `example/gtfs-static-and-rt.jv` --> Uses multiple endpoints from french NAP and showcases how to access GTFS data as well GTFS-RT data. Can be used to run the pipeline periodically (RT-data gets appended). This file is also used in my evaluation of the implementation-

Uses package [gtfs-realtime-binding](https://github.com/MobilityData/gtfs-realtime-bindings/tree/master) published under Apache2.0 license

Part of [RFC0006 GTFS-RT Support](https://github.com/jvalue/jayvee/tree/main/rfc/0006-gtfs-rt-support).

Implements following UACs of #219:
- [x] [UAC-3] gtfs-static-and-rt.jv is added to showcase the processing of GTFS-RT data as well as GTFS data.
- [x] [UAC-4] Every run of `gtfs-static-and-rt.jv` creates/updates ONE sqlite file
- [x] [UAC-5] Every run of `gtfs-static-and-rt.jv` downloads GTFS data and overwrites GTFS tables
- [x] [UAC-6] Every run of `gtfs-static-and-rt.jv` downloads GTFS-RT data and appends it to GTFS-RT tables
- [x] [UAC-7] Jayvee processes successfully `gtfs-static-and-rt.jv`

Please merge first #223 and #254 to main, before merging that one.

### CODE REVIEW

File: example/gtfs-rt-simple.jv [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+    block GTFSRTVehiclePositionFeedExtractor oftype HttpExtractor {
+		url: "https://proxy.transport.data.gouv.fr/resource/bibus-brest-gtfs-rt-vehicle-position";
+	}
+
+    block GTFSRTAlertFeedExtractor oftype HttpExtractor {
+		url: "https://proxy.transport.data.gouv.fr/resource/bibus-brest-gtfs-rt-alerts";
+	}
+
+    block GtfsRTTripUpdateInterpreter oftype GtfsRTInterpreter{
```
Comment (rhazn) [inline example/gtfs-rt-simple.jv:18]: Style: There should be a space before `{` (everywhere).

File: example/gtfs-rt-simple.jv [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        dropTable: false;
+	}
+
+    block AlertLoader oftype SQLiteLoader {
+		table: "gtfs-rt-alert";
+		file: "./gtfs.sqlite";
+        dropTable: false;
+	}
+
+    GTFSRTTripUpdateFeedExtractor
```
Comment (rhazn) [inline example/gtfs-rt-simple.jv:92]: Style: I think all non-source blocks should be all indented only once.

**Review by rhazn — CHANGES_REQUESTED** [review]
Also looks okay, also should be rebased to #254 so the changes to review are minimal.

File: example/gtfs-rt-simple.jv (hunk 1 above)
Comment (schlingling) [reply example/gtfs-rt-simple.jv]: changed

File: example/gtfs-rt-simple.jv (hunk 2 above)
Comment (schlingling) [reply example/gtfs-rt-simple.jv]: changed

**Review by rhazn — APPROVED** [review]

**Pull request merged**